### DISCUSSION TITLE
Zoom Members Practical 2 (series_02)

### DISCUSSION THREAD

**Discussion by ISchwandner** [IBAHCM/RPiR#425] [Old]
Hi all,

I thought it's best to start a new discussion for the 2nd Practical series as we are quite a large group and this may amke it easier to find things.

Hope you're all going well with the practical

**Comment by ISchwandner** [comment]
Hi Zoomers,

Here is the link to my repository for practical series 2:
[https://github.com/IBAHCM/ISchwandner_series02.git](url)

I would very much appreciate if some of you could try to access it and try to compile reports from my scripts, to see if it all works.

Thank you very much and have a good week!

**Reply by sreedevi-gummuluri** [reply]
I don't think I have access to your repository @ISchwandner 

Thanks,
Sreedevi

**Reply by ISchwandner** [reply]
Thanks Sreedevi,

Something appears to be wrong with the link, I can't open it either.

**Reply by ISchwandner** [reply]
Could you please try copying this link again? https://github.com/IBAHCM/ISchwandner_series02.git

**Reply by sreedevi-gummuluri** [reply]
Yes, this one is working! thank you.

**Comment by richardreeve** [comment]
@AilsaWatson - you should join this group since you're doing this online and put your repo in here.

**Comment by richardreeve** [comment]
It's not great that there are so many people in this Zoom group - this would work much better in person - but given how many you there are, please talk to each other to make sure that at least a couple of you check each repository.

**Comment by sreedevi-gummuluri** [comment]
Hi Everyone,

Here is the link to my repository for practical 2.  I'd greatly appreciate any feedback or comments.

https://github.com/IBAHCM/sreedevi-gummuluri_series02.git

Best,
Sreedevi

**Comment by maryadejoh** [comment]
Hello everyone.
Here is the link to my repository: https://github.com/IBAHCM/maryadejoh_series02.git

Kindly let me know if you encounter any issues. Thank you.

Mary

**Reply by maryadejoh** [reply]
Just to be sure, is it just the three of us having this discussion on the group?
I would like to know if you have access to the repositories of other members please. Thank you

Cheers,
Mary

**Reply by ISchwandner** [reply]
Hi Mary,

This discussion should be open to all members of the course, in case some people shift from Zoom to the classroom or the other way around. 

Except Sreedevi, you and me noone has shared their links here yet. But I just took the liberty to look up Dylan's repository in the list and checked it. For sure also Agnes was in our group. I'm not entirely sure who else. But that should be quite a few to get started with.

Hope this helps.

Imo
